pkgs/development/python-modules/playwright/default.nix

```diff
@@ -0,0 +1,247 @@
+, websockets
+
+# For building the browser bundle
+, cacert
+, chromium
+, ffmpeg
+, makeFontsConf
+}:
+
+let
+  inherit (stdenv.hostPlatform) system;
+  throwSystem = throw "Unsupported system: ${stdenv.hostPlatform.system}";
+
+  driverVersion = "1.25.0";
+
+  driver = let
+    suffix = {
+      x86_64-linux = "linux";
+      aarch64-linux = "linux-arm64";
+      x86_64-darwin = "mac";
+      aarch64-darwin = "mac-arm64";
+    }.${system} or throwSystem;
+    filename = "playwright-${driverVersion}-${suffix}.zip";
+  in stdenv.mkDerivation {
+    pname = "playwright-driver";
+    version = driverVersion;
+
+    src = fetchurl {
+      url = "https://playwright.azureedge.net/builds/driver/${filename}";
+      sha256 = {
+        x86_64-linux = "1nah45q9mihzkz2hqx6dwn7ac88vc22m5zspr23hc7dd1q4ll4ys";
+        aarch64-linux = "0wg4z2gb2fhvv4c2gjm6ncvbg9bkaavzq76x1bmm4lsnhjz312hp";
+        x86_64-darwin = "1nd6ll6dj0aqic3hh65y0br051ckbcap8spqbvzm19g8dn7vgv7m";
+        aarch64-darwin = "08qrzahmdhdv8flfjb58m27dqihq8jc4gwix6knc71209a9x8b83";
+      }.${system} or throwSystem;
+    };
+    sourceRoot = ".";
+
+    nativeBuildInputs = [ unzip ];
+    buildInputs = [ nodejs ];
+
+    postPatch = ''
+      # Use Nix's NodeJS instead of the bundled one.
+      substituteInPlace playwright.sh --replace '"$SCRIPT_PATH/node"' '"${nodejs}/bin/node"'
+      rm node
+
+      # Hard-code the script path to be our $out directory. The alternative would be to add
+      # coreutils to the closure and use realpath like the upstream script does.
+      substituteInPlace playwright.sh \
+        --replace 'SCRIPT_PATH="$(cd "$(dirname "$0")" ; pwd -P)"' "SCRIPT_PATH=$out"
+
+      patchShebangs playwright.sh package/bin/*.sh
+    '';
+
+    dontBuild = true;
+
+    installPhase = ''
+      runHook preInstall
+
+      mkdir -p $out/bin $out/share/playwright-driver
+      mv playwright.sh package $out/share/playwright-driver/
+      ln -s $out/share/playwright-driver/playwright.sh $out/bin/playwright
+
+      runHook postInstall
+    '';
+
+    passthru = {
+      inherit filename;
+    };
+  };
+
+  browsers-mac = stdenv.mkDerivation {
+    pname = "playwright-browsers";
+    version = driverVersion;
+
+    src = runCommand "playwright-browsers-base" {
+      outputHashMode = "recursive";
+      outputHashAlgo = "sha256";
+      outputHash = {
+        x86_64-darwin = "1nd6ll6dj0aqic3hh65y0br051ckbcap8spqbvzm19g8dn7vgv7m";
+      }.${system} or throwSystem;
+    } ''
+      export PLAYWRIGHT_BROWSERS_PATH=$out
+      ${driver}/playwright.sh install
+      rm -r $out/.links
+    '';
+
+    dontConfigure = true;
+    dontBuild = true;
+
+    installPhase = ''
+      mkdir $out
+      cp -r * $out/
+    '';
+
+    doCheck = false;
+  };
+
+  browsers-linux = let
+    fontconfig = makeFontsConf {
+      fontDirectories = [];
+    };
+  in runCommand "playwright-browsers" {
+    nativeBuildInputs = [
+      makeWrapper
+      pkgs.jq # We want jq and not pythonPackages.jq
```

Comment:
Please change that to an inherit where callPackage is used to not use pkgs in inputs.